cmp/grid/GridModel.js

```diff
@@ -219,11 +221,13 @@ export class GridModel {
         treeMode = false,
         showSummary = false,
         selModel,
-        stateModel = null,
         emptyText = null,
         sortBy = [],
         groupBy = null,
 
+        persistWith,
+        persistOptions,
```

Comment:
Do we still require separate options here? I thought we were folding persistOptions into persistWith?